src/expression/keywords.js

```diff
@@ -1,4 +1,4 @@
 // Reserved keywords not allowed to use in the parser
-export const keywords = {
-  end: true
-}
+export const keywords = new Set([
+  'end'
```

Comment:
Does this mean I can't use a variable called `end`?

Reply:
Correct, the keyword `end` is used is used to create a dynamic range for a matrix, like:

```js
const scope = { 
  A: [1, 2, 3, 4, 5] 
}
math.evaluate('A[3:end]', scope) // [3, 4, 5]
```

Reply:
I have opinions.

Not for this PR though.

Reply:
👍